client/components/DataManagement/DataManagementRow/index.jsx

```diff
@@ -854,18 +862,10 @@ const DataManagementRow = ({
                             currentDate
                         );
 
-                    const daysBetweenDates = checkTimeBetweenDates(
-                        currentDate,
-                        latestVersion.candidatePhaseReachedAt
-                    );
-                    const DAYS_TO_PROVIDE_FEEDBACK = 120;
                     const shouldShowAdvanceButton =
                         isAdmin &&
                         completedRequiredReports(latestVersion) &&
```

Comment:
Not a change in this PR, but this function prevents the button from appearing for a `CANDIDATE` test plan version, unless the reports required for a recommended test plan version has been met. Has that [requirement since changed](https://github.com/w3c/aria-at-app/issues/758#issuecomment-1699694393) again?

Reply:
OK, noted! Let me know if you need me to do anything about this line.

Reply:
We talked it over in a meeting! Thanks for clarifying what you meant Howard. Basically it should be possible to advance to recommended even without all the recommended reports completed. I will update this line to support that.